README.adoc

```diff
@@ -48,6 +48,7 @@ using, and contributing to Minishift:
 - https://docs.openshift.org/latest/minishift/getting-started/index.html[Getting started with Minishift]
 - https://docs.openshift.org/latest/minishift/using/index.html[Using Minishift]
 - https://docs.openshift.org/latest/minishift/contributing/index.html[Contributing to Minishift]
+- https://docs.openshift.org/latest/minishift/contributing/developing.html[Developing and building Minishift locally]
```

Comment:
```diff
- - https://docs.openshift.org/latest/minishift/contributing/index.html[Contributing to Minishift]
+ - https://docs.openshift.org/latest/minishift/contributing/index.html[Developing and contributing to Minishift]
```

Reply:
+1 In this case, we just update the link title, instead of adding a whole new link